.gitignore

```diff
@@ -14,7 +14,6 @@ build
 node_modules
 npm-debug.log
 ios
-android
```

Comment:
I'm not sure why it has been added here, probably because of RN (@HazAT?). Either we have to gitignore it or rename the folder, so idk what to do

Reply:
Hmm no idea either on why it is here but why do we need to remove it? Maybe we just change it to ./android to scope it to just that directory?